lte/gateway/python/magma/subscriberdb/store/sqlite.py

```diff
@@ -35,93 +38,131 @@ class SqliteStore(BaseStore):
     """
 
     def __init__(self, db_location, loop=None):
-        self._db_location = db_location
-        self._tlocal = threading.local()
+        self._db_location = self._create_db_locations(db_location)
         self._create_store()
         self._on_ready = OnDataReady(loop=loop)
 
-    @property
-    def conn(self):
-        """
-        Returns a thread local connection to the sqlite db.
-        """
-        if not getattr(self._tlocal, 'conn', None):
-            self._tlocal.conn = sqlite3.connect(self._db_location, uri=True)
-        return self._tlocal.conn
+
+    def _create_db_locations(self, db_location):
+        # db_location in subscriberdb.yml: file:/var/opt/magma/subscriber.db?cache=shared
```

Comment:
The yml config is confusing
Suggest changing this config to reflect that it is no longer a single file.
Maybe, change it to specify the folder

Reply:
make sense. I will specify the path in the subscriberdb.yml. if the file path is not included, it will default to in-memory db.